src/pages/home/Home.tsx

```diff
@@ -8,96 +8,100 @@ import useAuthentication from "../../hooks/useAuthentication";
 
 function Home() {
   const { userId } = useAuthentication();
-  const { data } = useGetUserStudentsQuery(userId!);
+  const { data: students } = useGetUserStudentsQuery(userId!);
   const navigate = useNavigate();
 
   return (
-    <table className="table">
-      <thead className="thead">
-        <tr>
-          <th>Filtros</th>
-          <th>DNI</th>
-          <th>Nombre</th>
-          <th>Apellido</th>
-          <th>Estado</th>
-          <th>Habilitado</th>
-          <th>Pago</th>
-          <th>Info</th>
-          <th>Eliminar/Dar Alta</th>
-        </tr>
-      </thead>
-      <tbody className="tbody">
-        {data &&
-          data.map((student) => (
-            <tr key={student.id}>
-              <td></td>
-              <td>{student.name}</td>
-              <td>{student.lastname}</td>
-              <td
-                style={{
-                  color: student.status === "A término" ? "#00bf63" : "#ff3131",
-                }}
-              >
-                <strong>{student.status}</strong>
-              </td>
-              <td>
-                <strong
-                  style={{ color: student.isActive ? "#00bf63" : "#ff3131" }}
+    <div className="students-container">
+      <table className="table">
+        <thead className="thead">
+          <tr>
+            <th>Filtros</th>
+            <th>DNI</th>
+            <th>Nombre</th>
+            <th>Apellido</th>
+            <th>Estado</th>
+            <th>Habilitado</th>
+            <th>Pago</th>
+            <th>Info</th>
+            <th>Eliminar/Dar Alta</th>
+          </tr>
+        </thead>
+        <tbody className="tbody">
+          {students &&
+            students.map((student) => (
+              <tr key={student.id}>
+                <td></td>
+                <td>{student.dni}</td>
+                <td>{student.name}</td>
+                <td>{student.lastname}</td>
+                <td
+                  style={{
+                    color:
+                      student.status === "A término" ? "#00bf63" : "#ff3131",
```

Comment:
Acá esto está mal, porque el status no es "A término" sino "En término", esto ocasiona que por más que esté en término el color sea rojo

<img width="173" height="83" alt="Image" src="https://github.com/user-attachments/assets/c96c7fd8-ac5c-4836-9574-dfab4f90212d" />


```suggestion
                      student.status === "En término" ? "#00bf63" : "#ff3131",
```